src/open_inwoner/kvk/tests/test_api.py

```diff
@@ -36,126 +36,82 @@ def test_search_endpoint(self):
 
         self.assertEqual(endpoint1, endpoint2)
 
-    def test_generic_search_by_kvk(self):
```

Comment:
Why is this removed?

Reply:
I removed it because it doesn't actually test anything: request calls are mocked, the bare search function contains no logic, hence the result is trivially equivalent to the mock. However, perhaps it's useful to keep even such a trivial test, in case someone further down the road tempers with the search method.